query/src/test/scala/filodb/query/exec/rangefn/AggrOverTimeFunctionsSpec.scala

```diff
@@ -306,4 +306,41 @@ class AggrOverTimeFunctionsSpec extends RawDataWindowingSpec {
     val aggregated = chunkedIt.map(x => (x.getLong(0), x.getDouble(1))).toList
     aggregated shouldEqual List((100000, 0.0), (120000, 2.0), (140000, 2.0))
   }
+
+  it("should correctly calculate holt winters") {
+    val positiveTrendData2 = Seq(15900.0, 15920.0, 15940.0, 15960.0, 15980.0, 16000.0)
+    val positiveTrendData3 = Seq(23850.0,23880.0,23910.0,23940.0,23970.0, 24000.0)
+    val positiveTrendData4 = Seq(31800.0, 31840.0, 31880.0, 31920.0, 31960.0, 32000.0) // 0+40x2000
```

Comment:
What is  0+40x2000?

Reply:
It is a prometheus example from their test cases. It is the exact test case.